libsolutil/FixedHash.h

```diff
@@ -48,6 +48,7 @@ class FixedHash
 
 	/// The size of the container.
 	enum { size = N };
+	static_assert(N != 0);
```

Comment:
I always find it funny that this works just as a statement top-level in a class :-). But it makes sense of course.